pkg/controllers/machine/disruption/drift.go

```diff
@@ -26,12 +26,20 @@ import (
 
 	"github.com/samber/lo"
 
+	"github.com/prometheus/client_golang/prometheus"
+
 	"github.com/aws/karpenter-core/pkg/apis/settings"
 	"github.com/aws/karpenter-core/pkg/apis/v1alpha5"
 	"github.com/aws/karpenter-core/pkg/cloudprovider"
+	"github.com/aws/karpenter-core/pkg/metrics"
 	"github.com/aws/karpenter-core/pkg/scheduling"
 )
 
+const (
+	ProvisionerStaticallyDrifted cloudprovider.DriftReason = "ProvisionerStaticallyDrifted"
+	NodeRequirementDrift         cloudprovider.DriftReason = "NodeRequirementDrifted"
```

Comment:
nit: good to keep the variable name the same
```suggestion
	NodeRequirementDrifted         cloudprovider.DriftReason = "NodeRequirementDrifted"
```